### ISSUE TITLE
questions about code 

### ISSUE THREAD

**Issue by vickersmith** [pangwong/pytorch-multi-label-classifier#6]
your multi_label_classifier was constructed by several concurrent FC layers .Another idea is that the label is encoded by one-hot encoder and add a sigmoid function after the 3rd FC layer.At last ,only a CrossEntropyLoss is used without any concurrent FC layers.Do you think the idea works or have you try it?Thanks very much!

**Comment by pangwong** [comment]
I have't tried it yet. In fact, the weights of single fc layer proposed in your idea and the weigths of multiple concurrent fc layers have the same parameters number as long as the size of feature map before fc is identical.
